### PR TITLE
Update the Bug Report Form for 2022 (#18275)2

### PR DESCRIPTION
**Pull request by karst**
This is to accommodate for new types of issues that could have been introduced with the New Save Format, Asset Packs, the ability to load in RCT Classic and lastly if the issue is related to zfighting.

If there's anything else that should be added, or things that should be changed let me know.

### CODE REVIEW

**Review by tupaschoal — APPROVED** [review]
It looks mostly fine for me, but I'll leave another team member to weigh in

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -18,31 +18,43 @@ body:
...
     description: You can see the version by running `openrct2 --version` or copy it from the About window. Open the "Disk and game options" menu (floppy disk on the top toolbar), choose "About 'OpenRCT2'" and click the copy button next to the version information.
   validations:
     required: true
 
-- type: textarea
+- type: dropdown
   attributes:
-    label: Describe the issue
-    description: Please describe the issue you are experiencing here.
+    label: Basegame
```
Comment (rik-smeets) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:28]: I don't think `basegame` is a real word. Below, you use the term `original game`. I think that one fits better here too.

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -18,31 +18,43 @@ body:
...
+    description: With what basegame do you have OpenRCT2 loaded? Did you test it with the other (select both if so)?
```
Comment (rik-smeets) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:29]: Replace here too.

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 3]
```diff
@@ -18,31 +18,43 @@ body:
...
     description: What things had an issue? Check all that apply.
     multiple: true
     options:
-      - New Save Format (NSF)
-      - The bug also exists in RCT2 (vanilla)
+      - The bug also exists in the original game (RCT2/RCTC)
       - This bug is specific to multiplayer
-      - Building the game
+      - This bug is a graphical glitch or error
+      - This issue has to do with parkobjects and/or asset packs
```
Comment (rik-smeets) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:46]: `parkobjects`-> please replace with `park objects`.

**Review by rik-smeets — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (karst) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:28]: Original game sounds more like if it occurs in what game even though neither is the game you are playing. Base game is what game you have chosen as game to load OpenRCT2 up with. I'll gladly split the words of, I think it's the Dutch in me combine words :P

Comment (rik-smeets) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:28]: Alright, I agree. Just split the words :)

**Pull request merged**